### PR TITLE
Return JSON error in case of malformed request body json in RPC API

### PR DESCRIPTION
**Pull request by i-skrypnyk**

### CODE REVIEW

**Comment by i-skrypnyk** [comment]
@MarcoOl94 @JcBernack please review, you were suggested by github :)

**Review by JcBernack — APPROVED** [review]
LGTM

File: sdk/src/main/scala/io/horizen/account/api/http/route/AccountEthRpcRejectionHandler.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  def rejectionHandler: RejectionHandler =
+    RejectionHandler.newBuilder()
+      .handle {
+        case MalformedRequestContentRejection(msg, _) =>
+          SidechainApiResponse(
+            EthJsonMapper.serialize(
+              new RpcResponseError(
+                new RpcId(),
+                RpcError.fromCode(RpcCode.ParseError, msg)
```
Comment (i-Alex) [inline sdk/src/main/scala/io/horizen/account/api/http/route/AccountEthRpcRejectionHandler.scala:20]: here we return error code `ParseError(-32700, "Parse error")`
infura example from Cronic returns `{"jsonrpc":"2.0","error":{"code":-32600,"message":"invalid json request"}}` which in our case is `InvalidRequest(-32600, "Invalid request")`

Let's double check what is actually expected.

**Review by i-Alex — CHANGES_REQUESTED** [review]

File: sdk/src/main/scala/io/horizen/account/api/http/route/AccountEthRpcRejectionHandler.scala (hunk 1 above)
Comment (i-skrypnyk) [reply sdk/src/main/scala/io/horizen/account/api/http/route/AccountEthRpcRejectionHandler.scala:20]: Decided to go with -32700, as it's description fully represent this case. Notified Cronicc about it.
image

**Review by i-Alex — APPROVED** [review]

**Review by paolocappelletti — APPROVED** [review]

**Pull request merged**